web/help/GuideUnit1.md

```diff
@@ -542,16 +573,17 @@ more hints, expand the sections below for an example and explanation.
     ~~~~~
 
     This type signature tells you that `thickCircle` needs two arguments,
-    both numbers, and the result is a picture.  Here's an example of a
-    program that uses `thickCircle`.
+    both of them being numbers, and the result is a picture.  
+    
+    Here's an example of a program that uses `thickCircle`.
 
     ~~~~~ . clickable
     program = drawingOf(pic)
     pic = thickCircle(5, 1)
     ~~~~~
 
     Functions beginning with "thick" draw shapes with a thick line, and
-    the thickness of that line is the last argument to the function.  So
+    the thickness of that line is the last argument to the function.
     The first argument is the radius of the circle, and the second is
```

Comment:
It isn't clear here that the second sentence is only talking about `thickCircle`.